### PR TITLE
Update afton.json

### PR DESCRIPTION
**Pull request by faris0x**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] You have completed your website. 
- [X] The website is reachable.  
- [  ] You're not using Vercel or Netlify.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by priyazsh — CHANGES_REQUESTED** [review]

File: domains/afton.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
     "owner": {
-        "username": "WilliamAfton-codes",
+        "username": "aftoncodes",
         "email": "williamafton.codes@gmail.com",
-        "twitter": "manlydalorian"
+        "twitter": "aftoncodes"
     },
     "record": {
-        "CNAME": "williamafton-codes.github.io"
+        "CNAME": "aftoncodes.vercel.app"
```
[the remark this answers is not in the document]
Comment (aftoncodes) [reply domains/afton.json:10]: Multiple subdomains are regsitered in the project with a Vercel deployment, I don't think it should be a problem

[the remark this answers is not in the document]
Comment (oyepriyansh) [reply domains/afton.json:10]: show me any?

[the remark this answers is not in the document]
Comment (aftoncodes) [reply domains/afton.json:10]: @oyepriyansh  frd.json, vishalkrsharma.json, probir-sarkar.json, mxnuchim.json, tdh.json, haru.json, mert.json, hyro.json, chloe.json, i2rys.json, yuphie.json, kapoor.json, chaman.json, nassro.json, matt3o0.json, fordevs.json, albin.json, mehmetabak.json, lalmanthakur.json, habib.json, osada.json, phelix.json, yasser.json, krinzal.json, scor32k.json, snowyid.json, ccortina.json, davidpatricio.json, wahid7852.json, imskanand.json, madhavgupta.json, ohqte.json, isqne.json, abdeba.json, dhruva.json, tanmay.json, yassinm.json, romidev.json, megalith.json, anderson.json, vulcanwm.json, elijah629.json, haridarshan.json, hung-contact.json, deekshasharma.json, snowy.json and hung.json

[the remark this answers is not in the document]
Comment (oyepriyansh) [reply domains/afton.json:10]: Those are **URL** records, which indicate redirects.
image

 You can verify that their domains are redirecting to a domain like something.vercel.app. If you want to set up redirects, you can use URL records. If you want to add this domain to your website, you can utilize services like [GitHub Pages](https://github.io), [Cloudflare Pages](https://pages.dev), or [Railway](https://railway.app).

**Review by mtgsquad — APPROVED** [review]

**Pull request merged**